### PR TITLE
chore(ci): include snippets folder into tfhe npm package

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -463,6 +463,7 @@ build_web_js_api_parallel: install_rs_check_toolchain install_wasm_pack
 		-- --features=boolean-client-js-wasm-api,shortint-client-js-wasm-api,integer-client-js-wasm-api,parallel-wasm-api,zk-pok \
 		-Z build-std=panic_abort,std && \
 	find pkg/snippets -type f -iname workerHelpers.worker.js -exec sed -i "s|from '..\/..\/..\/';|from '..\/..\/..\/tfhe.js';|" {} \;
+	jq '.files += ["snippets"]' tfhe/pkg/package.json > tmp_pkg.json && mv -f tmp_pkg.json tfhe/pkg/package.json
```
Comment (IceTDrinker) [inline Makefile:466]: isn't there an in place mode for jq ?

**Review by IceTDrinker — COMMENTED** [review]
Looks good for the principle, I believe jq has an in place mode to modify files

File: Makefile (hunk 1 above)
Comment (IceTDrinker) [reply Makefile:466]: also maybe a && ?

Comment (soonum) [reply Makefile:466]: Nope, no in-place mode for jq sadly.

**Review by IceTDrinker — APPROVED** [review]

**Review by IceTDrinker — APPROVED** [review]
Thanks a lot !

**Review by IceTDrinker — APPROVED** [review]

**Pull request merged**